src/index.ts

```diff
@@ -65,7 +67,7 @@ export class Marqeta {
   user: UserApi
   card: CardApi
   cardProduct: CardProductApi
-
+  webHooks: WebhooksApi
```

Comment:
Typically, we have a blank line _after_ the ivars, and before the constructor... like before this PR.